### DISCUSSION TITLE
Heads Up:  Security issue posted to Arch Newsfeed regarding xz potentially impacting docker and podman

### DISCUSSION THREAD

**Discussion by joekm** [netbrain/zwift#78] [General]
Link is here:  
https://archlinux.org/news/the-xz-package-has-been-backdoored/

Maybe not  a concern for us but, seems prudent to at least give the heads-up.

**Comment by quietvoid** [comment]
The image runs Debian 12 "Bookworm", so it should have `xz-utils` version `5.4.1` and thus should not be affected.
